gamemode/languages/sh_spanish.lua

```diff
@@ -2,35 +2,91 @@
 NAME = "Español"
 
 LANGUAGE = {
+	helix = "Helix",
+
+	introTextOne = "fist industries presenta",
+	introTextTwo = "en colaboración con %s",
+	introContinue = "pulsa espacio para continuar",
+
+	helpIdle = "Selecciona una categoría",
+	helpCommands = "Los parámetros de comandos con <flechas> son requeridos, mientras que los [corchetes] son opcionales.",
+	helpFlags = "Las Flags con un fondo verde son accesibles por este personaje.",
+
+	creditSpecial = "Muchas Gracias",
+	creditLeadDeveloper = "Desarrollador Principál",
+	creditUIDesigner = "Diseñador de interfaz de Usuario",
+	creditManager = "Gerente del Proyecto",
+	creditTester = "Tester Principal",
+
+	chatTyping = "Escribiendo...",
+	chatTalking = "Hablando...",
+	chatYelling = "Gritando...",
+	chatWhispering = "Susurrando...",
+	chatPerforming = "Actuando...",
+	chatNewTab = "Nueva pestaña...",
+	chatReset = "Reiniciar posición",
+	chatResetTabs = "Reiniciar pestañas",
+	chatCustomize = "Customizar...",
+	chatCloseTab = "Cerrar Pestaña",
+	chatTabName = "Nombre de Pestaña",
+	chatAllowedClasses = "Clases de chat Permitidas",
+	chatTabExists = "¡Una pestaña de chat con ese nombre ya existe!",
+	chatMarkRead = "Marcar todos como leído",
+
+	community = "Comunidad",
+	checkAll = "Marcar Todos",
```

Comment:
In the context of the chatbox, this isn't selecting, but checking the values (aka, a true or false value):
![image](https://user-images.githubusercontent.com/7958349/47959685-97632a80-dfea-11e8-94b3-db5f37661355.png)
I believe in this case this translation would be wrong.

Reply:
Seleccionar is still better than Marcar, since you're selecting what chat categories you want. The definition of "marcar" is to add a note (or a mark) to identify or tell smth apart from smth, or to give information. In this case, the system offers you the available categories, and you're to select which ones you want and which ones you don't. Another possible translation would be "Elegir" (to choose).